src/Files.App/ViewModels/SidebarViewModel.cs

```diff
@@ -6,230 +6,133 @@
 using Files.App.Extensions;
 using Files.App.Filesystem;
 using Files.App.Helpers;
-using Files.App.UserControls;
 using Files.Backend.Services.Settings;
 using Files.Shared.EventArguments;
 using Files.Shared.Extensions;
 using Microsoft.UI.Dispatching;
-using Microsoft.UI.Xaml;
 using Microsoft.UI.Xaml.Controls;
 using Microsoft.UI.Xaml.Media.Imaging;
 using System;
 using System.Collections.Generic;
+using System.Collections.Immutable;
 using System.Collections.Specialized;
 using System.IO;
 using System.Linq;
+using System.Runtime.Versioning;
 using System.Threading.Tasks;
 using System.Windows.Input;
 
 namespace Files.App.ViewModels
 {
-	public class SidebarViewModel : ObservableObject, IDisposable
+	[SupportedOSPlatform("Windows10.0.17763")]
+	public class SidebarViewModel : ObservableObject, ISidebarViewModel
 	{
-		private IUserSettingsService UserSettingsService { get; } = Ioc.Default.GetRequiredService<IUserSettingsService>();
+		private readonly IUserSettingsService userSettingsService = Ioc.Default.GetRequiredService<IUserSettingsService>();
 
-		public ICommand EmptyRecycleBinCommand { get; private set; }
-
-		private IPaneHolder paneHolder;
+		private readonly DispatcherQueue dispatcherQueue;
 
+		private readonly ImmutableArray<SectionType> sectionOrder = new SectionType[]
+		{
+			SectionType.Home,
+			SectionType.Favorites,
+			SectionType.Library,
+			SectionType.Drives,
+			SectionType.CloudDrives,
+			SectionType.Network,
+			SectionType.WSL,
+			SectionType.FileTag,
+		}.ToImmutableArray();
+
+		private IPaneHolder? paneHolder;
 		public IPaneHolder PaneHolder
 		{
-			get => paneHolder;
+			get => paneHolder!;
 			set => SetProperty(ref paneHolder, value);
 		}
 
-		public IFilesystemHelpers FilesystemHelpers => PaneHolder?.FilesystemHelpers;
-
-		private DispatcherQueue dispatcherQueue;
-
-		public BulkConcurrentObservableCollection<INavigationControlItem> SideBarItems { get; init; }
-
-		public static readonly GridLength CompactSidebarWidth = SidebarControl.GetSidebarCompactSize();
+		public ICommand EmptyRecycleBinCommand { get; }
 
 		private NavigationViewDisplayMode sidebarDisplayMode;
-
 		public NavigationViewDisplayMode SidebarDisplayMode
 		{
 			get => sidebarDisplayMode;
 			set
 			{
 				if (SetProperty(ref sidebarDisplayMode, value))
-				{
 					OnPropertyChanged(nameof(IsSidebarCompactSize));
-					UpdateTabControlMargin();
```

Comment:
Why is this method removed here?

Reply:
This method updates a property that is never read. The property has been removed.